src/process.ts

```diff
@@ -117,7 +118,8 @@ export class ApprovalProcess<S extends Schema> {
    * Get all approval process definitions registered to sobjects
    */
   async list() {
-    const res = await this._conn.request('/process/approvals');
+    // TODO fix type
+    const res = await this._conn.request<any>('/process/approvals');
```

Comment:
I see several of these `TODO fix type` comments. Is there any reason we can't fix them now?

Reply:
I have no idea what these APIs return. I can go digging.